maths/find_unique.py

```diff
@@ -0,0 +1,22 @@
+def find_unique_xor(nums: list) -> float:
+    """
+    Find unique from a given list of numbers with 2n+1 entries of which n element appears twice and there is one unique element.
```

Comment:
```suggestion
    Find unique from a given list of numbers with 2n+1 
    entries of which n element appears twice and there 
    is one unique element.
```
This will fix your failing checks